com.ca.lsp.cobol/lsp-core-cobol-parser/src/main/java/com/ca/lsp/core/cobol/model/ErrorCode.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package com.ca.lsp.core.cobol.model;
+
+/**
+ * This enum represents the error codes that are used to determine some special type of errors. See
+ * the instance documentation for more details.
+ */
+public enum ErrorCode {
+  /** This copybook does not present in the workspace */
```

Comment:
I will be more specific, like ''..in the predefined copybook folder"

Reply:
The "predefined" is too strict here, I would prefer just "...in the copybook folder"